polus-tiledtiff-converter-plugin/README.md

```diff
@@ -48,7 +57,5 @@ _Note:_ All tile sizes must be a positive integer than is a multiple of 16
 ```bash
 docker run -v /path/to/data:/data labshare/polus-tiledtiff-converter-plugin \
```

Comment:
Is the version necessary ? ex. `labshare/polus-tiledtiff-converter-plugin:{version}`

Reply:
This is a really good comment, and it made me think about this more. What we should be doing is making sure the most recent version of our container has a `latest` tag in addition to the `{version}`. That way, if someone tries to run this as it, it will work just fine.

I'll have to include that in the next cookiecutter template. For now though, we should include a version number.